content/src/service/caching/CacheManager.ts

```diff
@@ -0,0 +1,71 @@
+import { EntityType } from 'dcl-catalyst-commons'
+import { CacheByType } from './Cache'
+
+/** This manger is used to create cache instances, based on some pre-configured defaults */
+export class CacheManager {
+  private readonly cacheSizes: Map<string, number>
+
+  constructor(cacheSizes: Map<string, number> = new Map()) {
+    this.cacheSizes = new Map(Array.from(cacheSizes.entries()).map(([name, size]) => [name.toUpperCase(), size]))
+  }
+
+  buildEntityTypedCache<K, V>(cacheConfig: CacheConfig): CacheByType<K, V> {
```

Comment:
why do you need `K` and `V` here?

Reply:
It's because you want to receive a `CacheByType<K, V>`. So you need to pass them to `buildEntityTypedCache`